src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -500,7 +500,7 @@ public function isDirectoryEmpty($directory, $hidden = false)
     public function isDirectoryNotEmpty($directory, $hidden = false)
     {
         return $this->isDirectory($directory)
```

Comment:
could this not be written as

```
return ! $this->isDirectoryEmpty();
```

Reply:
Now that I can think about it more clearly I think it should return an exception if it's not a directory.

Reply:
What i meant was that the `isDirectoryNotEmpty` method doesn't need to repeat the logic, it can just use the `isDirectoryEmpty` method and negate the result.

Then you don't need to have almost the same logic in two different methods.